docs/design/ovs-pipeline.md

```diff
@@ -53,62 +52,129 @@
+|               | bit  22     |                           | 0b1           | L7NPRedirectRegMark             | Indicating L7 Antrea NetworkPolicy disposition of redirect.                              |
+|               | bits 21-22  | OutputRegField            | 0b01          | OutputToOFPortRegMark           | Output packet to an OVS port.                                                            |
+|               |             |                           | 0b10          | OutputToControllerRegMark       | Send packet to Antrea Agent.                                                             |
+|               | bits 25-32  | PacketInOperationField    | 0b00000001    | PacketInNPLoggingOperation      | Indicating packet needs logging for NetworkPolicy packetIn operation.                    |
+|               |             |                           | 0b00000010    | PacketInNPRejectOperation       | Indicating packet should be rejected for NetworkPolicy packetIn operation.               |
+|               |             |                           | 0b00000100    | PacketInNPStoreDenyOperation    | Indicating the corresponding connection has been dropped or rejected.                    |
+| NXM_NX_REG1   | bits 0-31   | TargetOFPortField         |               |                                 | Egress OVS port of packet.                                                               |
+| NXM_NX_REG2   | bits 0-31   | SwapField                 |               |                                 | Swap values in flow fields in OpenFlow actions.                                          |
+|               |             | PacketInTableField        |               |                                 | OVS table where the packet is decided to be sent to controller.                          |
+| NXM_NX_REG3   | bits 0-31   | EndpointIPField           |               |                                 | Field where stores selected Service Endpoint IPv4 address.                               |
+|               | bits 0-31   | APConjIDField             |               |                                 | Field to store Conjunction ID for Antrea Policy.                                         |
+| NXM_NX_REG4   | bits 0-15   | EndpointPortField         |               |                                 | Field stores IPv4 address of a Service's selected Endpoint.                              |
+|               | bits 16-18  | ServiceEPStateField       | 0b001         | EpToSelectRegMark               | Packet needs to do Service Endpoint selection.                                           |
+|               | bits 16-18  | ServiceEPStateField       | 0b010         | EpSelectedRegMark               | Packet has done Service Endpoint selection.                                              |
```

Comment:
is it `EpToSelectRegMark` instead of `EpToSelectedRegMark`?

Reply:
I didn't find `EpToSelectedRegMark`.